maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/service/PassportAssessmentService.java

```diff
@@ -31,6 +32,13 @@ public PassportAssessmentDTO update(UpdatePassportAssessment updatePassportAsses
         PassportAssessmentDTO passportAssessmentDTO =
                 passportAssessmentMapper.updatePassportAssessmentToPassportAssessmentDTO(updatePassportAssessment);
         log.info("Updating existing passport assessment record");
+        PassportAssessmentEntity existingPassportAssessmentEntity = passportAssessmentImpl.find(passportAssessmentDTO.getId());
```

Comment:
This check is redundant as it is already being done in one of the validator classes.